docs/OV_Runtime_UG/deployment/local-distribution.md

```diff
@@ -47,47 +47,45 @@ As shown in the picture above, some plugin libraries may have OS-specific depend
+      |    Device    |       Dependency        |                      Location                         |
+      +==============+=========================+=======================================================+
+      |     CPU      |            —            |                          —                            |
+      +--------------+-------------------------+-------------------------------------------------------+
+      |     GPU      | | OpenCL.dll            | | ``C:\Windows\System32\opencl.dll``                  |
+      |              | | cache.json            | | ``.\runtime\bin\intel64\Release\cache.json``   or   |
+      |              |                         | | ``.\runtime\bin\intel64\Debug\cache.json``          |
+      +--------------+-------------------------+-------------------------------------------------------+
+      |     GNA      |         gna.dll         | | ``.\runtime\bin\intel64\Release\gna.dll``    or     |
+      |              |                         | | ``.\runtime\bin\intel64\Debug\gna.dll``             |
+      +--------------+-------------------------+-------------------------------------------------------+
+      |  Arm® CPU    |            —            |                          —                            |
+      +--------------+-------------------------+-------------------------------------------------------+
 
    .. tab-item:: Linux OS
 
-      .. list-table::
-         :header-rows: 1
-
-         * - Device
-           - Dependency
-         * - CPU
-           - ``-``
-         * - GPU
-           - ``libOpenCL.so``, ``cache.json``
-         * - GNA
-           - ``gna.dll``
-         * - Arm® CPU
-           - ``-``
+      +--------------+-------------------------+-------------------------------------------------------+
+      |    Device    |       Dependency        |                      Location                         |
+      +==============+=========================+=======================================================+
+      |     CPU      |            —            |                          —                            |
+      +--------------+-------------------------+-------------------------------------------------------+
+      |     GPU      | | libOpenCL.so          | | ``/usr/lib/x86_64-linux-gnu/libOpenCL.so.1``        |
+      |              | | cache.json            | | ``./runtime/lib/intel64/cache.json``                |
+      +--------------+-------------------------+-------------------------------------------------------+
+      |     GNA      |      libgna.so          | ``./runtime/lib/intel64/libgna.so.3``                 |
+      +--------------+-------------------------+-------------------------------------------------------+
+      |  Arm® CPU    |            —            |                          —                            |
+      +--------------+-------------------------+-------------------------------------------------------+
 
    .. tab-item:: MacOS
```

Comment:
may be we need to add explicitly macOS arm64 and macOS x86_64 ? The same for Linux
Because it's a little bit strange to see GNA and ARM CPU in the same table.